site/app/templates/forum/createThread.twig

```diff
@@ -0,0 +1,49 @@
+{% for css in page_css %}
+    <link rel="stylesheet" href="{{ base_url }}/{{ css }}" />
+{% endfor %}
+
+{% for js in page_js %}
+    <script type="text/javascript" language="javascript" src="{{ base_url }}/{{ js }}"></script>
+{% endfor %}
```

Comment:
Don't inject JS/CSS files directly (I know this was how they were before, and that's a bug) as we end up in a mess with how browsers are caching these files and how we might try and invalidate that cache. It's why there's the `Output::addInternal{Js,Css}` and `Output::addVendor{Js,Css}` methods which add a datestring to the end of the file to help properly invalidate a file on update and force a re-cache.